deps/chakrashim/src/v8isolate.cc

```diff
@@ -241,7 +241,7 @@ bool Isolate::IdleNotification(int idle_time_in_ms) {
 }
 
 void Isolate::LowMemoryNotification() {
-  CHAKRA_UNIMPLEMENTED();
+  IsolateShim::GetCurrent()->CollectGarbage();
```

Comment:
CI failed with the following error- need jsrt::IsolateShim here?

```
linux/nodes/ubuntu1404-64/out/Release/obj.target/chakrashim/deps/chakrashim/src/v8isolate.o ../deps/chakrashim/src/v8isolate.cc
../deps/chakrashim/src/v8isolate.cc:244:3: error: use of undeclared identifier 'IsolateShim'; did you mean 'jsrt::IsolateShim'?
  IsolateShim::GetCurrent()->CollectGarbage();
```

Reply:
Yes namespace must be missing. will update the PR. Thanks for the reviews!